apps/backend/package.json

```
@@ -20,13 +20,16 @@
   },
   "dependencies": {
     "@payloadcms/db-mongodb": "3.45.0",
+    "@payloadcms/email-nodemailer": "^3.47.0",
     "@payloadcms/next": "3.45.0",
```

please use the correct version (should be 3.45.0) and also pin the dependency

this requires you to update pnpm lock. 

```suggestion
    "@payloadcms/db-mongodb": "3.45.0",
    "@payloadcms/email-nodemailer": "3.45.0",
    "@payloadcms/next": "3.45.0",
```